dokku

```diff
@@ -132,7 +132,11 @@ case "$1" in
 
         # if we can't post-deploy successfully, kill new container
         kill_new() {
-          docker inspect $id &> /dev/null && docker stop $id > /dev/null && docker kill $id > /dev/null
+          docker inspect $id &> /dev/null \
+          && docker stop $id > /dev/null \
+          && docker kill $id > /dev/null \
+          && docker rm $id > /dev/null
```

Comment:
Can someone comment on whether this is the right approach? `;` instead of `&&` might be better in this case. That way stop, kill and rm are called regardless of their previous failing. (Does `docker stop` fail if the container was already stopped?)